caddyhttp/proxy/upstream.go

```diff
@@ -22,7 +22,7 @@ import (
 )
 
 var (
-	supportedPolicies = make(map[string]func() Policy)
+	supportedPolicies = make(map[string]func(string) Policy)
```

Comment:
Per @mholt's recommendation in #1463 it's just `string`, but changing it to `...string` would be fairly trivial.